RELEASE_NOTES.md

```diff
@@ -15,6 +15,9 @@
   `frequenz.api.common.location.Location` is being used instead. The `Location`
   message from the common API also has a `country_code` member.
 
+- The following gRPC methods have been renamed:
+  `StreamComponentData` -> `SubscribeComponentData`
```

Comment:
Looks like a number of methods have been renamed, should they also be mentioned here?

Reply:
It was a mistake, fixed.